modules/support-starting-an-alternative-image-with-toolbox.adoc

```diff
@@ -6,7 +6,13 @@
 [id="starting-an-alternative-image-with-toolbox_{context}"]
 = Starting an alternative image with `toolbox`
 
+ifndef::openshift-origin[]
 By default, running the `toolbox` command starts a container with the `registry.redhat.io/rhel8/support-tools:latest` image. You can start an alternative image by creating a `.toolboxrc` file and specifying the image to run.
+endif::openshift-origin[]
+
+ifndef::openshift-origin[]
+By default, running the `toolbox` command starts a container with the `quay.io/openshift/origin-support-tools:latest` image. You can start an alternative image by creating a `.toolboxrc` file and specifying the image to run.
+endif::openshift-origin[]
```

Comment:
@peer-review-squad Existing text ^^. I updated only the image name in the first sentence and the ifdefs.